ui/main/src/app/services/cards/server/CardsServer.ts

```
@@ -14,7 +14,7 @@ import {CardCreationReportData} from '../model/CardCreationReportData';
 import {FieldToTranslate} from '@ofServices/cards/model/FieldToTranslate';
 import {Observable} from 'rxjs';
 import {ServerResponse} from '../../../business/server/serverResponse';
-import {LightCard} from 'app/model/LightCard';
+import {Card} from 'app/model/Card';
```

Biome (1.9.4)

[error] Shouldn't redeclare 'Card'. Consider to delete it or rename it.

'Card' is defined here:


(lint/suspicious/noRedeclare)